wp-content/themes/pub/wporg-learn-2024/patterns/contribute-page-content.php

```diff
@@ -0,0 +1,145 @@
+<?php
+/**
+ * Title: Contribute Page Content
+ * Slug: wporg-learn-2024/contribute-page-content
+ * Inserter: no
+ */
+
+?>
+ 
+<!-- wp:heading {"style":{"typography":{"fontSize":"36px","fontStyle":"normal","fontWeight":"400"}},"fontFamily":"eb-garamond"} -->
+<h2 class="wp-block-heading has-eb-garamond-font-family" style="font-size:36px;font-style:normal;font-weight:400">
+	<?php esc_html_e( 'Contribute', 'wporg-learn' ); ?>
+</h2>
+<!-- /wp:heading -->
```

Comment:
Should be the h1, and I've just updated the h1 styles so you should simply be able to do this now:
```suggestion
<!-- wp:heading {"level":1} -->
	<h1 class="wp-block-heading"><?php esc_html_e( 'Contribute', 'wporg-learn' ); ?></h1>
<!-- /wp:heading -->
```

Reply:
![Screenshot 2024-06-20 at 11 49 34 AM](https://github.com/WordPress/Learn/assets/1017872/66bc72d4-1395-4357-9856-9dd28a096e7c)